releasetasks/templates/final_verify.yml.tmpl

```diff
@@ -0,0 +1,49 @@
+{% set buildername = "{}_final_verify".format(branch) %}
+-
+    taskId: "{{ stableSlugId(buildername) }}"
+    reruns: 5
+    task:
+        provisionerId: aws-provisioner-v1
+        workerType: b2gtest
+        created: "{{ now }}"
+        deadline: "{{ now.replace(hours=6) }}"
+        expires: "{{ never }}"
+        priority: "high"
+        retries: 5
+        scopes:
+
```

Comment:
It worked because the image doesn't require any scopes. Also no caches used. Probably it's OK until we switch to an in-tree image. Can you just remove the "scopes:" line then?